pkgs/development/interpreters/php/8.1.nix

```diff
@@ -2,8 +2,8 @@
 
 let
   base = callPackage ./generic.nix ((removeAttrs _args [ "fetchpatch" ]) // {
-    version = "8.1.27";
-    hash = "sha256-oV/XPqRPLfMLB9JHhuB9GUiw6j7tC4uEVzXVANwov/E=";
+    version = "8.1.28";
+    hash = "sha256-i+RQCW4BU8R9dThOfdWVzIl/HVPOAGBwjOlYm8wxQe4=";
     extraPatches = [
```

Comment:
The patch can now be removed.

Reply:
Done.
(Even noticed the patch, but assumed it's still needed given that it still applies, but the heuristic was wrong this time I guess)